Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y fonts-liberation libappindicator3-1 libasound2 libatk-bri
 RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
     dpkg -i google-chrome-stable_current_amd64.deb && \
     apt-get -fy install && \
-    wget https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_linux64.zip && \
+    wget https://chromedriver.storage.googleapis.com/76.0.3809.68/chromedriver_linux64.zip && \
```

Comment:
We have to match this to our chrome version; see https://github.com/publiclab/plots2/pull/5525#issuecomment-486043530

Reply:
So that should be ok...